lib/kubot/main.rb

```diff
@@ -1,11 +1,17 @@
 module Kubot
+  SLACK_CLIENT_ID = ENV["SLACK_CLIENT_ID"]
+  SLACK_CLIENT_SECRET = ENV["SLACK_CLIENT_SECRET"]
+  SLACK_SUPPORT_TEAM = ENV["SLACK_SUPPORT_TEAM"]
   class Main < Clamp::Command
-
     subcommand "start", "Start the kubot" do
-      option "--token", "TOKEN", "Slack api token",
-        :environment_variable => "SLACK_API_TOKEN", :required => true
       def execute
-        Kubot::Bot.run
+        if SLACK_CLIENT_ID != nil && SLACK_CLIENT_SECRET != nil && SLACK_SUPPORT_TEAM != nil 
+          Setup.run
+          run Auth.run!
+        else
+          puts "ERROR: you need to set 3 environment variables:\nSLACK_CLIENT_ID\nSLACK_CLIENT_SECRET\nSLACK_SUPPORT_TEAM"
```

Comment:
It's always better to be more user-friendly, if some variable is empty, just print that exactly this environment variable is not defined/empty.